src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java

```diff
@@ -171,27 +59,4 @@ public void onPlayerAdvancementDone(final PlayerAdvancementDoneEvent event) {
                 .hoverEvent(descriptionComponent
                     .color(frameSpecificColor))));
     }
```

Comment:
This only works with vanilla (or normal cusotm) advancements im guessing (onPlayerAdvancementDone) is a bukkit event

Reply:
yeah, that's probably it. but I can't find something in the UltimateAdvancementAPI that lets you add a listener for when any custom advancement is granted